__tests__/app.test.js

```diff
@@ -0,0 +1,43 @@
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const app = require('../app')
+const testData = require('../db/data/test-data/')
+const request = require('supertest')
+
+beforeEach(() => seed(testData))
+afterAll(() => db.end())
+
+describe('/api/topics', () => {
+    describe('GET - status 200 - responds with an array of topic objects', () => {
+        test('responds with an array of objects', () => {
+            return request(app)
+                .get('/api/topics')
+                .expect(200)
+                .then(({ body: { topics } }) => {
+                    expect(topics).toHaveLength(3)
+                    topics.forEach(topic => expect(typeof topic).toBe('object'))
+                })
+        })
+        test('each object in returned array has `slug` and `description` properties', () => {
+            return request(app)
+                .get('/api/topics')
+                .expect(200)
+                .then(({ body: { topics } }) => {
+                    topics.forEach(topic => {
```

Comment:
This is all good, but for the future I recommend the use of the following flexible object matchers:

.toMatchObject
.objectContaining